zulip/integrations/zephyr/zephyr_mirror_backend.py

```diff
@@ -1108,14 +1100,13 @@ def die_gracefully(signal, frame):
         logger.error("\n" + "nagios_path is required with nagios_class\n")
         sys.exit(1)
 
-    zulip_account_email = options.user + "@mit.edu"
-    import zulip
+    zulip_account_email = options.zulip_email + "@mit.edu"
```

Comment:
This is wrong; the old `--user` option was a username, not an email address, which this line makes clear.  

I think we could probably just do a split to construct the username.

Reply:
@timabbott: Hey, if `zulip_account_email` is already an email now, should we add `@mit.edu` at the end? Can't we just stop doing that? Or will this integration only be used by people at MIT?